### DISCUSSION TITLE
Move discussions to overall organization (.github repository)

### DISCUSSION THREAD

**Discussion by jdsika** [lichtblick-suite/lichtblick#282] [Ideas]
We should move the discussions to the .github repository in order to have them visible here:
![grafik](https://github.com/user-attachments/assets/0db78977-6acc-4c67-8fbf-5e19996c7645)

We could have categories for different repositories if necessary but I believe we should have one central discussion platform for the overall GitHub organization.

**Comment by carolinabamorim** [comment]
Hi @jdsika , thank you for your suggestion. It will be considered.
